docs/our_work/epma_autocoding.md

```diff
@@ -0,0 +1,62 @@
+---
+title: 'ePMA Auto-Coding'
+summary: 'A rule-base data processing pipeline to map free text medication descriptions to SNOMED codes using the dm+d standard.'
+origin: ''
+tags: ['PYTHON','SECONDARY CARE','PRESCRIBING','TEXT DATA']
```

Comment:
could this not be considered to fall under the linkage tag? it's linkage of medications

also might fit into data validation project type tag, could add that

also could you add a tag on whether the project is WIP, COMPLETE or ON PAUSE